### PR TITLE
refactor: Fix backend integration

### PR DESCRIPTION
**Pull request by ziemen4**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the `SwapForm` component to manage dynamic exchange rates and enforce output limits.
	- Added a user feedback mechanism for insufficient funds in the `SwapFormItem` component.

- **Bug Fixes**
	- Improved calculation logic for converted amounts and output limits.

- **Documentation**
	- Updated comments for clarity in the configuration settings.

### CODE REVIEW

File: src/components/SwapForm/Form.tsx [hunk 1]
```diff
@@ -103,27 +125,37 @@ export default function SwapForm() {
             return;
         }
 
-        // Calculate input converted amount (rate * input amount)
-        const inputConverted = parseFloat(amount) * EXCHANGE_RATE;
+        // Calculate input converted amount (rate * input amount) (assumes 1 USDC = 1 USD)
+        const inputConverted = parseFloat(amount) * 1;
```
[the remark this answers is not in the document]
Comment (alexhooketh) [reply src/components/SwapForm/Form.tsx:129]: > if USDC and USDT are assumed to be pegged to USD, consider adding comments to clarify this assumption and ensure consistency.

cc @ziemen4

File: src/components/SwapForm/Form.tsx [hunk 2]
```diff
@@ -103,27 +125,37 @@ export default function SwapForm() {
...
-            // Calculate output converted amount (rate * output amount)
-            const outputConverted = output * EXCHANGE_RATE;
+            // Calculate output converted amount (rate * output amount) in USD (assumes 1 USDT = 1 USD)
+            const outputConverted = output * 1;
             setOutputConvertedAmount(`$${outputConverted.toFixed(2)}`);
         }
     };
 
-    const isSwapDisabled = !inputAmount || !tronAddress || insufficientFunds; // Disable swap if no amount, no Tron address, or insufficient funds
+    const isSwapDisabled = !inputAmount || !tronAddress || insufficientFunds || maxOutputSurpassed; // Disable swap if no amount, no Tron address, or insufficient funds, or max output surpassed
```
[the remark this answers is not in the document]
Comment (alexhooketh) [reply src/components/SwapForm/Form.tsx:158]: @ziemen4 wdyt

[the remark this answers is not in the document]
Comment (ziemen4) [reply src/components/SwapForm/Form.tsx:159]: I dont see why not

**Comment by alexhooketh** [comment]
@ziemen4 do I merge this

**Comment by ziemen4** [comment]
> @ziemen4 do I merge this

Yeah it builds, go for it

**Review by alexhooketh — APPROVED** [review]

**Pull request merged**